### DISCUSSION TITLE
Mi práctica pokédex

### DISCUSSION THREAD

**Discussion by OscarPedroza** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#482] [Show and tell]
Buenas noches compañeros, les comparto mi práctica de la pokédex, aunque no soy muy fan de pokemon fue muy divertido realizar la esta práctica.
- [Link Pokédex](https://oscarpedroza.github.io/PracticaPokedex.github.io/)
- [Link repositiorio](https://github.com/OscarPedroza/PracticaPokedex.github.io)
Saludos!

**Comment by SujeiRodriguez** [comment]
XD el psyduck...

**Reply by OscarPedroza** [reply]
Jajaja, tenía la expresión adecuada XD

**Comment by JLuisPrz** [comment]
Se ve muy bien, te quedo genial.

**Reply by OscarPedroza** [reply]
Muchas gracias compañero!

**Comment by EslebanPerez** [comment]
Super bien 👍 Me gusto mucho la animación!

**Reply by OscarPedroza** [reply]
Muchas gracias compañero!

**Comment by HectorRamirezN** [comment]
te quedo muy bien me gusto el diseño y si quieres puedes poner el peso en kg y la altura en metros, por ejemplo pikachu tiene altura de 4 si lo divides entre 10 da 0.4m

**Reply by OscarPedroza** [reply]
Muchas gracias por el consejo!, lo tendré en cuenta para actualizarlo.

**Comment by axlgoze** [comment]
Hola, Te quedó muy padre tu  pokédex 👏👏, me gustó mucho la forma que hiciste la parte de JS y me apoye mucho de esa parte para terminar mi práctica, me parecio muy ordenado, sencillo y claro MUCHAS GRACIAS por compartir tu trabajo ✌

**Comment by AndresAVargas** [comment]
Te quedo genial compañero, disculpa hiciste una función para cada movimiento o hay una forma de sacar todos los movimiento con una sola funcion?

**Reply by OscarPedroza** [reply]
Muchas gracias compañero, así es utilice una función para los movimientos almacenándolos en un array e imprimiendo cada uno en un lista con un for.
